### PR TITLE
Adding ACPI for Phone (1)

### PR DESCRIPTION
**Pull request by index986**
### What Changed
Adding ACPI for Phone (1)

### Checklist

* [x] Is what you changed Tested?                (Required)
* [ ] Is all the ACPI Source based on your DTS?
* [x] Are the Correct Compiler Versions used?    (Required)
* [x] Are there no References from the Compiler? (Required)

### CODE REVIEW

File: Platforms/Xiaomi/lavender/DSDT.dsl
Comment (Robotix22) [inline Platforms/Xiaomi/lavender/DSDT.dsl:1]: Please Remove this File if the other .asl File is Used for Compiling.

**Review by Robotix22 — CHANGES_REQUESTED** [review]

File: Platforms/Xiaomi/lavender/DSDT.dsl
Comment (index986) [reply Platforms/Xiaomi/lavender/DSDT.dsl:1]: Sorry but I dont really understand what do you mean under " other .asl File "

Comment (Robotix22) [reply Platforms/Xiaomi/lavender/DSDT.dsl:1]: In the spacewar Folder are two Files that can be Compiled
The First One is .asl and the other one is .dsl
You have to Remove the one you didn't use for making the .aml File

Comment (index986) [reply Platforms/Xiaomi/lavender/DSDT.dsl:1]: Fixed

**Review by Robotix22 — APPROVED** [review]

**Pull request merged**